Application/EdFi.Admin.DataAccess/Repositories/ClientAppRepo.cs

```diff
@@ -199,6 +199,16 @@ public async Task<ApiClient> GetClientAsync(string key)
             }
         }
 
+        public ApiClient GetClientIncludingApplication(string key)
```

Comment:
This new method only includes the necessary fields for the `Change Secret` button.
More specifically, the issue gets fixed by not including the `ApplicationEducationOrganizations`.